internal/namespaces/baremetal/v1alpha1/custom_server_create_test.go

```diff
@@ -35,7 +35,15 @@ func Test_CreateServer(t *testing.T) {
 				core.TestCheckGolden(),
 				core.TestCheckExitCode(0),
 			),
-			AfterFunc:   waitAndDeleteServerAfterFunc,
+			AfterFunc: func(ctx *core.AfterFuncCtx) error {
```

Comment:
why not use `waitServerAfter("Server")` and `deleteServer("Server")` in this file ?